docs/SignalProvider.md

```diff
@@ -0,0 +1,28 @@
+# SignalProvider
+
+`SignalProvider` uses Signaler to tell an app what A/B tests are active.
+
+## Usage
+
+`SignalProvider` should wrap the root element or at least the largest subtree of your app that needs A/B testing.
+
+```jsx
+React.render(
+  <SignalProvider>
+    <KillerAbTestedApp/>
+  </SignalProvider>
+);
+
+```
+
+Nothing will be rendered until signals are retrieved.
```

Comment:
Maybe allow an optional default component to be rendered? Either that or lifecycle-esque hooks to enable someone to do that themselves. lifecycle sounds like overkill to me, but if it could be done without much code, it'd be pretty cool. could be as simple as calling a function prop when its ready.